src/components/Workspace.js

```diff
@@ -10,19 +10,26 @@ import ns from '../config/css-ns';
  * @memberof Workspace
  * @private
  */
-const Workspace = ({ windows }) => (
-  <div className={ns('workspace')}>
-    {
-      Object.values(windows).map(window => (
-        <ConnectedWindow
-          id={window.id}
-          key={window.id}
-          window={window}
-        />
-      ))
-    }
-  </div>
-);
+export class Workspace extends React.Component {
+  /**
+   * guess what
```

Comment:
Can we leave out that docstring? I think we should only add docstrings that add real value, I don't think this would qualify :-)
Probably required some quorom on the eslint settings, maybe an exception for commong stuff like `render` and the React lifecycle hooks would be good?

Reply:
👍

Reply:
Sorry, just wanted to tease the useless docstrings like /** Renders things */

Reply:
I think a PR updating the config here: https://github.com/ProjectMirador/mirador/blob/master/.eslintrc#L15-L23 would be most welcome, especially based on our now shared experience.